lib/nixd/src/Server.cpp

```diff
@@ -160,6 +161,54 @@ void Server::addDocument(lspserver::PathRef File, llvm::StringRef Contents,
            [](std::shared_ptr<EvalDraftStore::EvalResult> Result) {});
 }
 
+CompletionHelper::Items
+CompletionHelper::fromStaticEnv(const nix::SymbolTable &STable,
+                                const nix::StaticEnv &SEnv) {
+  Items Result;
+  for (auto [Symbol, Displ] : SEnv.vars) {
```

Comment:
`Symbol` is used as a ref while `Displ` is unused?

Reply:
They can be trivially copied (actually they are integers)